client/client-multi/client-cpp/impl/utils/ThreadPool.cpp

```diff
@@ -50,29 +47,21 @@ void Worker::operator ()()
         auto task = threadPool_.tasks_.front();
         threadPool_.tasks_.pop_front();
 
-        KAA_MUTEX_UNLOCKING("tasksGuard_");
         KAA_UNLOCK(tasksLock);
-        KAA_MUTEX_UNLOCKED("tasksGuard_");
 
         try {
             task();
-        } catch (...) {
-            KAA_LOG_WARN("Caught exception while executing thread pool task");
-        }
+        } catch (...) {}
```

Comment:
Empty exception handler is not recommended. Any exceptional situation must be reported.